### PR TITLE
1549 ally fiori guidelines

### LINKED ISSUE #1549: Evaluate Fiori keyboard interactions
Evaluate which component is used by Luigi and which Fiori keyboard interactions can be applied.
https://cxwiki.sap.com/display/luigi/Fiori+Keyboard+Interaction

**OUTCOME:**
Important:
- All popovers (dropdowns) should be closed one ESC click (we have 7 of them in Shellbar)
- No access to the full left side navigation with Tab key press. Only to the nested list
- No TAB Navigation access (TabNav.html), page 174 of the guidelines
- First page load Tab-focus priority. For now it jumps into the iframe skipping the left side nav and skiping half of the shellbar elements

Nice to have:
- No access to button in Modal (Miscellaneous2 modal only) but main key buttons work fine
- Interactive Modal with actions/forms inside: 
       1. CTRL+ENTERR should trigger default action to finish/finalize a task. In our case removes the title of Modal. 
Especially helpful for wizard dialogs with optional steps. The optional steps can be skipped by using CTRL+ENTER. 
      2.ALT+Enter: Switch focus between master and details section in a Master/Details application. 
Alternative: Show a properties dialog which displays meta information about the focused UI element or item. 

**Outcome:** https://cxwiki.sap.com/display/luigi/Fiori+Keyboard+Interaction

### PR DESCRIPTION
**Pull request by UlianaMunich**
Fixes #1549 

A tiny fix to make accessible the burger menu via Tab keypress in semiCollapsible mode

### CODE REVIEW

File: core/src/navigation/TopNav.html [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+  class="fd-shellbar {hideNavComponent ? 'hideNavComponent' : ''}"
+  tabindex="0"
+>
   <div class="fd-shellbar__group fd-shellbar__group--product">
     {#if responsiveNavSetting==="simple" || responsiveNavSetting==="simpleMobileOnly" || responsiveNavSetting==="Fiori3"}
-    <span
-      class="sap-icon--menu2 lui-burger"
+    <button
+      class="fd-shellbar__button fd-button sap-icon--menu2 lui-burger"
+      aria-controls="0AcWE812"
```
Comment (stanleychh) [inline core/src/navigation/TopNav.html:11]: Just curious, how does `0AcWE812` come from?

Comment (UlianaMunich) [reply core/src/navigation/TopNav.html:11]: @stanleychh have no idea, from the git log I see it was added by @JohannesDoberer  6 months ago, or maybe it was a merge from another branch.

Comment (UlianaMunich) [reply core/src/navigation/TopNav.html]: @there is no ID bonded to that aria-controls so I removed 2 of them from the TopNav.html 
@stanleychh please check and approve the changes if all is good :)

**Review by stanleychh — APPROVED** [review]
LGTM

**Review by JohannesDoberer — APPROVED** [review]

**Pull request merged**